pkg/operator/credentialsrequest/status_test.go

```diff
@@ -184,14 +184,6 @@ func TestClusterOperatorStatus(t *testing.T) {
 			cloudPlatform:      configv1.GCPPlatformType,
 			expectedConditions: []configv1.ClusterOperatorStatusCondition{},
 		},
-		{
-			name:              "upgradable false if parent cred removed",
-			credRequests:      []minterv1.CredentialsRequest{},
-			parentCredRemoved: true,
```

Comment:
the parentCredRemoved field from the test cases struct now appears to be unused.